app/javascript/articles/components/__tests__/SaveButton.test.jsx

```diff
@@ -3,17 +3,31 @@ import { fireEvent, render } from '@testing-library/preact';
 import { axe } from 'jest-axe';
 import { SaveButton } from '../SaveButton';
 
+it('should not show button when post owned by current user', () => {
```

Comment:
```suggestion
it('should not show bookmark button when saveable is false', () => {
```
just a suggestion, since `SaveButton` doesn't have knowledge of current user etc